CHANGELOG.md

```diff
@@ -11,17 +11,18 @@ Package  | Version  | Link
 ---|---|---
 SDK Core| v1.0.3 | [symbol-sdk](https://www.npmjs.com/package/symbol-sdk)
 Catbuffer | v1.0.1 | [catbuffer-typescript](https://www.npmjs.com/package/catbuffer-typescript)
-Client Library | v1.0.2  | [symbol-openapi-typescript-fetch-client](https://www.npmjs.com/package/symbol-openapi-typescript-fetch-client)
+Client Library | v1.0.3  | [symbol-openapi-typescript-fetch-client](https://www.npmjs.com/package/symbol-openapi-typescript-fetch-client)
 
 - fix: replaced `instanceof` statements. These statements are problematic when npm installs the dependency in multiples modules. 
+- feat: added mosaic revocation support.
 
 ## [1.0.2] - 25-Oct-2021
 
 **Milestone**: Symbol Mainnet
 Package  | Version  | Link
 ---|---|---
 SDK Core| v1.0.2 | [symbol-sdk](https://www.npmjs.com/package/symbol-sdk)
-Catbuffer | v1.0.1 | [catbuffer-typescript](https://www.npmjs.com/package/catbuffer-typescript)
+Catbuffer | v1.0.0 | [catbuffer-typescript](https://www.npmjs.com/package/catbuffer-typescript)
```

Comment:
why did you lower this to 1.0.0? package.json is referencing 1.0.1

Reply:
I'm correcting a mistake in the CHANGELOG.md

SDK 1.0.2 -> Catbuffer 1.0.0 
SDK 1.0.3 (To be released) -> Catbuffer 1.0.1  

https://github.com/symbol/symbol-sdk-typescript-javascript/blob/v1.0.2/package.json#L96